doc/source/testing.rst

```diff
@@ -66,13 +80,19 @@ Run all functional tests.
 
 .. code-block:: bash
 
-    $ tox -t functional
+    $ tox -e $(tox -l | grep functional | paste -d, -s -)
```

Comment:
Same:
```suggestion
    $ tox -e functional
```

Reply:
Sorry but it would execute functional only on the current python version, see https://github.com/ansible/molecule/issues/1764#issuecomment-465992799